graph/path/yen_ksp.go

```diff
@@ -5,16 +5,17 @@
 package path
 
 import (
-	"math"
 	"sort"
 
 	"gonum.org/v1/gonum/graph"
 	"gonum.org/v1/gonum/graph/iterator"
 )
 
 // YenKShortestPaths returns the k-shortest loopless paths from s to t in g.
-// YenKShortestPaths will panic if g contains a negative edge weight.
-func YenKShortestPaths(g graph.Graph, k int, s, t graph.Node) [][]graph.Node {
+// with path costs no greater than cost relative to the shortest path.
```

Comment:
I find the word "relative" here misleading as it evokes a ratio, a division, but there's none. I had to check the code and the tests to understand. Maybe replace it with "compared", or something like "with path cost differences no greater than cost compared to the shortest path"?

Reply:
I was thinking relative as opposed to absolute.

Reply:
SGTM